src/lua/httpc.c

```diff
@@ -155,15 +155,36 @@ luaT_httpc_request(lua_State *L)
 	if (req == NULL)
```

Comment:
Regarding the documentation request. AFAIU, the change also affects `:post()` helper, not only `:request()`.

Reply:
Right. I'll add.